platform/golden/provisioners/00_create_manifest/roles/packages/vars/main.yml

```diff
@@ -0,0 +1,157 @@
+---
```

Comment:
this looks like a rebase gone wrong. Probably as a result of my force push error to main...